database/migrations/2024_05_27_040232_create_user_service_packages_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('user_service_packages', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('user_id')->constrained('users')->cascadeOnUpdate()->cascadeOnDelete();
+            $table->foreignId('service_package_id')
+                ->constrained('service_packages')
+                ->cascadeOnUpdate()
+                ->cascadeOnDelete();
+            $table->timestamp('start_date');
```

Comment:
không có status hả em ?.

Reply:
dạ tụi em tính được từ start_date ạ

Reply:
em đang mua gói này 10 lần:
- task hãy lấy danh sách hết package đã hết hạn.
- task hãy lấy package đang activate
- task get all lịch sử thanh toán package và phân biệt đâu là package đang còn hạn và đâu là package đã hết hạn
- task, đọc 1 cuốn truyện có gói package tương tứng.

em viết query theo điều kiện trên giúp anh thử nha.